src/resources/SecurityCache/tests/SecurityCache.spec.ts

```diff
@@ -19,6 +19,7 @@ describe('securityCache', () => {
 
     describe('list', () => {
         const providerId = 'PROVIDER_ID';
+        const securityCacheFilters = {filteringTerm: 'test', filteringMode: 'PREFIX', providerIds: [providerId]};
```

Comment:
for my understanding what are the other possible values of filteringMode ?

Reply:
It can be `Prefix` or `Substring`. You don't have to bother with that for the moment, it will use the default value of the service.